src/components/Tutorial/NavigationBar.vue

```diff
@@ -117,11 +126,26 @@ export default {
   },
   computed: {
     pageSections() {
-      const tutorialPath = this.$route.path.replace(/#$/, '');
-      return this.tutorialState.linkableSections.map(section => ({
-        ...section,
-        path: `${tutorialPath}#${section.anchor}`,
-      }));
+      const projectReferences = this.chapters
```

Comment:
I think it could help to create a new computed property (that depends on `chapters` and `identifierUrl`) which returns the the current project from the chapters list representing this tutorial, which can just be used here as `project.sections` (etc)